client.go

```diff
@@ -220,13 +228,15 @@ func WithRuntime(name string) NewContainerOpts {
 	}
 }
 
+// WithSnapshotter sets the provided snapshotter for use by the container
```

Comment:
I don't think so, the NewCcontainerOpts kinda makes that clear with the type system